backend/config/config.go

```diff
@@ -80,7 +80,8 @@ type Config struct {
 	} `mapstructure:"extension"`
 
 	DataReport struct {
-		Key string `mapstructure:"key"`
+		Key           string `mapstructure:"key"`
```

Comment:
建议为MachineIDFile字段添加注释，说明其用途和默认值，以提高代码可读性。

```suggestion
// MachineIDFile 指定存储机器ID的文件路径，默认为"/app/static/.machine_id"
MachineIDFile string `mapstructure:"machine_id_file"`